README.md

```diff
@@ -24,7 +24,7 @@ NGINX Prometheus exporter makes it possible to monitor NGINX or NGINX Plus using
 page](https://nginx.org/en/docs/http/ngx_http_stub_status_module.html#stub_status). [NGINX
 Plus](https://www.nginx.com/products/nginx/) provides a richer set of metrics via the
 [API](https://nginx.org/en/docs/http/ngx_http_api_module.html) and the [monitoring
-dashboard](https://www.nginx.com/products/nginx/live-activity-monitoring/). NGINX Prometheus exporter fetches the
+dashboard](https://docs.nginx.com/nginx/admin-guide/monitoring/live-activity-monitoring/). NGINX Prometheus exporter fetches the
 metrics from a single NGINX or NGINX Plus, converts the metrics into appropriate Prometheus metrics types and finally
 exposes them via an HTTP server to be collected by [Prometheus](https://prometheus.io/).
```

Comment:
```suggestion
dashboard](https://docs.nginx.com/nginx/admin-guide/monitoring/live-activity-monitoring/). NGINX Prometheus exporter
fetches the metrics from a single NGINX or NGINX Plus, converts the metrics into appropriate Prometheus metrics types
and finally exposes them via an HTTP server to be collected by [Prometheus](https://prometheus.io/).
```

@aknot242 a small tweak is needed to fix the markdownlint warning, once this is in one of us will merge.